api/src/bigquerytest/java/org/pmiops/workbench/cohorts/CohortMaterializationServiceTest.java

```diff
@@ -62,6 +92,14 @@ public void testMaterializeCohortOneMale() {
     assertThat(response.getNextPageToken()).isNull();
   }
 
+  @Test
+  public void testMaterializeCohortICD9Group() {
```

Comment:
Should we add another one of these for a demographics-based cohort?

Reply:
The only types that have group/intermediate lookups are icd9/icd10. Doesn't apply to Demographics.